packages/belos/tpetra/test/BlockGmres/CMakeLists.txt

```diff
@@ -1,19 +1,32 @@
 
 
+ASSERT_DEFINED(Anasazi_SOURCE_DIR)
 ASSERT_DEFINED(${PACKAGE_NAME}_ENABLE_Triutils)
+ASSERT_DEFINED(Tpetra_INST_FLOAT)
+
 IF (${PACKAGE_NAME}_ENABLE_Triutils)
+  IF (Tpetra_INST_FLOAT)
```

Comment:
I don't think any of the Dashboard builds test `float`, alas, so you may have to enable it locally to get test coverage.